tests/Unit/CssInlinerTest.php

```diff
@@ -3140,6 +3140,90 @@ public function inlineCssNotInDebugModeIgnoresOnlyInvalidExcludedSelector(): voi
         self::assertStringContainsString('<p class="z"></p>', $result);
     }
 
+    /**
+     * @test
+     *
+     * @dataProvider excludedCssRuleDataProvider
+     *
+     * @param string $css
+     * @param array<string> $exludedSelectors
+     * @param string $expectedInlineCss
+     */
+    public function addExcludedCssSelectorCanExcludeCssSelectors(string $css, array $exludedSelectors, string $expectedInlineCss): void
```

Comment:
There's a coding style line length limit of 120 characters.

```suggestion
    public function addExcludedCssSelectorCanExcludeCssSelectors(
        string $css,
        array $exludedSelectors,
        string $expectedInlineCss
    ): void
```